### PR TITLE
SamyGO.PVRDecoder version 1.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109541&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm seeing this:
image
image
image

File: manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: SamyGO.PVRDecoder
```
Comment (Trenly) [inline manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.installer.yaml:4]:
```suggestion
PackageIdentifier: SamyGo.PVRDecoder
```
Must match existing publisher casing or there will be Git conflicts

File: manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SamyGO.PVRDecoder
```
Comment (Trenly) [inline manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: SamyGo.PVRDecoder
```
Must match existing publisher casing or there will be Git conflicts

File: manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: SamyGO.PVRDecoder
```
Comment (Trenly) [inline manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.yaml:4]:
```suggestion
PackageIdentifier: SamyGo.PVRDecoder
```
Must match existing publisher casing or there will be Git conflicts

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.yaml (hunk 3 above)
Comment (upintheairsheep) [reply manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.yaml:4]: SamyGO is correct name.

Comment (stephengillie) [reply manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.yaml:4]: An Internet search shows both letters of GO are capitalized. The other manifests will need to be updated to match this one.

**Comment by stephengillie** [comment]
> I'm seeing this:  image

Same:
image

File: manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.yaml (hunk 3 above)
Comment (Trenly) [reply manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.yaml:4]: > An Internet search shows both letters of GO are capitalized. The other manifests will need to be updated to match this one.

In that case, the old manifests will need to be *completely removed*, wait for the publish pipeline to publish all changes removing the SamyGo manifests, then they will need to all be added back individually with the new SamyGO name.

Doing it any other way will result in directory conflicts

**Comment by upintheairsheep** [comment]
Appears to be a false positive for Windows Defender, out of our control

**Comment by stephengillie** [comment]
> Appears to be a false positive for Windows Defender, out of our control

We can recheck occasionally. I'm willing to retest a package once a day, as long as there are only a few packages needing retesting.

File: manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.installer.yaml:4]: Please update file name and path to match this change.

File: manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.locale.en-US.yaml (hunk 2 above)
Comment (stephengillie) [reply manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.locale.en-US.yaml:4]: Please update file name and path to match this change.

File: manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.yaml (hunk 3 above)
Comment (stephengillie) [reply manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.yaml:4]: Pending https://github.com/microsoft/winget-pkgs/issues/111198

**Comment by stephengillie** [comment]
The only item remaining that's preventing merging is the ISV name mismatch, pending:
* https://github.com/microsoft/winget-pkgs/issues/111198

**Comment by BrandonWanHuanSheng** [comment]
Very clear information about ./samyGOPVRDecoder.exe: files.

Verified:       Unsigned
        Link date:      5:43 AM 14/2/2014
        Publisher:      n/a
        Company:        United Key Crackers
        Description:    SamyGO PVR-Decoder
        Product:        SamyGO PVR-Decoder by LordByte in 2014
        Prod version:   1.5.3.4
        File version:   1.5.3.4
        MachineType:    32-bit

What do you think about the PackageVersion:?
Because Sigcheck did not detect that version is 1.0. Where did you found 1,0?
http://download.samygo.tv/PC%20Applications/samyGOPVRDecoder.zip 
InstallerSHA256: 923D1EBC760A0309482B144259006E4B205EBA1B6860C3C197E8F1F932CD9A05

**Comment by upintheairsheep** [comment]
> Very clear information about ./samyGOPVRDecoder.exe: files.
> 
> Verified: Unsigned Link date: 5:43 AM 14/2/2014 Publisher: n/a Company: United Key Crackers Description: SamyGO PVR-Decoder Product: SamyGO PVR-Decoder by LordByte in 2014 Prod version: 1.5.3.4 File version: 1.5.3.4 MachineType: 32-bit
> 
> What do you think about the PackageVersion:? Because Sigcheck did not detect that version is 1.0. Where did you found 1.0? http://download.samygo.tv/PC%20Applications/samyGOPVRDecoder.zip InstallerSHA256: 923D1EBC760A0309482B144259006E4B205EBA1B6860C3C197E8F1F932CD9A05

It’s good. I am currently on vacation for the next week so I will only be maintaining these pulls to prevent them from being closed. Thank you.

**Comment by stephengillie** [comment]
> Very clear information about ./samyGOPVRDecoder.exe: files.
> 
> Verified: Unsigned Link date: 5:43 AM 14/2/2014 Publisher: n/a Company: United Key Crackers Description: SamyGO PVR-Decoder Product: SamyGO PVR-Decoder by LordByte in 2014 Prod version: 1.5.3.4 File version: 1.5.3.4 MachineType: 32-bit
> 
> What do you think about the PackageVersion:? Because Sigcheck did not detect that version is 1.0. Where did you found 1.0? http://download.samygo.tv/PC%20Applications/samyGOPVRDecoder.zip InstallerSHA256: 923D1EBC760A0309482B144259006E4B205EBA1B6860C3C197E8F1F932CD9A05

1.5.3.4 is reported by the application, but Apps and Features shows 1.0:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**